repos/spack_repo/builtin/packages/ucc/package.py

```diff
@@ -58,6 +58,15 @@ def configure_args(self):
         args = []
         args.extend(self.with_or_without("cuda", activation_value="prefix"))
         args.extend(self.with_or_without("nccl", activation_value="prefix"))
+
+        if "+cuda" in self.spec:
+            if self.spec.variants["cuda_arch"].values != ("none",):
+                gencode_args = [
+                    f"-gencode=arch=compute_{arch},code=sm_{arch}"
```

Comment:
You should probably use `self.cuda_flags()`, inherited from CudaPackage.

Reply:
Good catch! My bad, I wanted to check if there was any utility for that and I completely forgot. Thanks!